content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
+
+Jika URL akses dari klaster yang dipulihkan diubah dari klaster sebelumnya, server API Kubernetes harus dikonfigurasi ulang dengan sesuai dengan itu. Pada kasus ini, restart server API Kubernetes dengan flag `--etcd-servers=$NEW_ETCD_CLUSTER` bukan flag `--etcd-servers=$OLD_ETCD_CLUSTER`. Ganti `$NEW_ETCD_CLUSTER` dan `$OLD_ETCD_CLUSTER` dengan alamat IP masing-masing. Jika load balancer digunakan di depan klaster etcd, kamu mungkin perlu memperbarui penyeimbang beban sebagai gantinya.
+
+Jika mayoritas anggota etcd telah gagal secara permanen, klaster etcd dianggap gagal. Dalam skenario ini, Kubernetes tidak dapat membuat perubahan apa pun ke kondisi saat ini. Meskipun _Pod_ terjadwal mungkin terus berjalan, tidak ada _Pod_ baru yang bisa dijadwalkan. Dalam kasus seperti itu, pulihkan klaster etcd dan kemungkinan juga untuk mengkonfigurasi ulang server API Kubernetes untuk memperbaiki masalah ini.
+
+## Meng-upgrade dan memutar kembali klaster etcd
+
+Pada Kubernetes v1.13.0, etcd2 tidak lagi didukung sebagai backend penyimpanan untuk klaster Kubernetes baru atau yang sudah ada. Timeline untuk dukungan Kubernetes untuk etcd2 dan etcd3 adalah sebagai berikut:
+
+- Kubernetes v1.0: etcd2 only
+- Kubernetes v1.5.1: etcd3 support added, new clusters still default to etcd2
+- Kubernetes v1.6.0: new clusters created with `kube-up.sh` default to etcd3,
+                     and `kube-apiserver` defaults to etcd3
+- Kubernetes v1.9.0: deprecation of etcd2 storage backend announced
+- Kubernetes v1.13.0: etcd2 storage backend removed, `kube-apiserver` will
+                      refuse to start with `--storage-backend=etcd2`, with the
+                      message `etcd2 is no longer a supported storage backend`
+
+Sebelum memutakhirkan v1.12.x kube-apiserver menggunakan `--storage-backend=etcd2` ke
+v1.13.x, data etcd v2 harus dimigrasikan ke backend penyimpanan v3 dan
+permintaan kube-apiserver harus diubah untuk menggunakan `--storage-backend=etcd3`.
+
+Proses untuk bermigrasi dari etcd2 ke etcd3 sangat tergantung pada bagaimana
+klaster etcd diluncurkan dan dikonfigurasi, serta bagaimana klaster Kubernetes diluncurkan dan dikonfigurasi. Kami menyarankan kamu berkonsultasi dengan dokumentasi penyedia kluster kamu untuk melihat apakah ada solusi yang telah ditentukan.
+
+Jika cluster kamu dibuat melalui `kube-up.sh` dan masih menggunakan etcd2 sebagai penyimpanan backend, silakan membaca [Kubernetes v1.12 etcd cluster upgrade docs](https://v1-12.docs.kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#upgrading-and-rolling-back-etcd-clusters)
+
+## Masalah umum: client balancer etcd dengan secure endpoint
```

Comment:
Can we translate "client balancer etcd" into "penyeimbang klien etcd"?
```suggestion
## Masalah umum: client balancer etcd dengan _secure endpoint_
```

Reply:
I agree, I think  "penyeimbang klien etcd" is better translation.